flowzone.yml

```diff
@@ -3183,6 +3183,8 @@ jobs:
 
     env:
       DOCKER_BUILDKIT: "1"
+      # renovate: datasource=github-releases depName=trufflesecurity/trufflehog
+      TRUFFLEHOG_VERSION: "3.91.0"
```

Comment:
Move this env to the step, it does not apply to any other steps in the job (avoid global envs)